gsoc_contributors_guide.md

```diff
@@ -5,9 +5,74 @@ The Echoshader team aims to recruit talented [Google Summer of Code (GSoC)](http
 
 ## Getting Started
 
-TBD
+As this project is an ongoing development of echopype, there are a couple of preliminary items you should become familiar
+with before diving in. We outline these items below as well as provide some additional helpful resources. 
 
+### Storage format
 
+In echopype, we utilize two methods for storing results: 
+
+* netcdf files - the current defacto file for working with array-oriented scientific data from climate research.
+Although it is not necessary to understand the netCDF library in its entirety, Unidata does provide a well documented 
+[netcdf](https://unidata.github.io/netcdf4-python/) python interface. This documentation describes useful aspects of 
+how netcdf defines common terms such as groups, dimensions, variables, and attributes.  
+
+* [Zarr](https://zarr.readthedocs.io/en/stable/) - a format for the storage of chunked, compressed, N-dimensional 
+arrays. Zarr has similar characteristics to netcdf, but has the added benefit of being able to be both read and wrote to
+concurrently from multiple threads or processes. Zarr provides a great overview of its [storage specifications](https://zarr.readthedocs.io/en/stable/spec/v2.html#hierarchies)
```

Comment:
I don't think it's necessary to go into concurrency/threads/processes here. Just the chunking part and some comments about cloud-native will suffice.